apps/web/components/walkthrough/Walkthrough.tsx

```diff
@@ -17,11 +19,32 @@ interface WalkthroughProps {
   startingSectionId: StartingSectionIdType;
 }
 
+export type WalkthroughAnswerType =
+  | string
+  | string[]
+  | Record<string, string>
+  | undefined;
```

Comment:
Do we want to actually handle undefined here with the answer type since an undefined answer doesn't make sense we could handle it at a higher level? 

We may have to do this (handle undefined in this type); not sure right now with my understanding of the code.

Reply:
Yeah we'll see. It's currently because of I'm tracking state but since I'm going to redo that part at a higher level I was waiting to see what comes out in the wash.